docs/overview/streaming.md

```diff
@@ -0,0 +1,51 @@
+---
+id: overview_streaming
+title: "Streaming"
+---
+
+Zio-elastic search is streaming friendly library and there are few specific APIs that are used for creating ZIO streams. 
```

Comment:
Suggestion:
"ZIO Elasticsearch is a streaming friendly library offering a few different API methods for creating ZIO streams out of search results.